docs/get_started/getting_started_python.md

```diff
@@ -64,33 +64,36 @@ $ python -m pip install tf-nightly
 
 ```shell
 # Also include -DIREE_BUILD_TENSORFLOW_COMPILER=ON if you want the TF compiler.
-$ cmake -G Ninja -B ../iree-build/ \
+$ cd iree-build/  # Make and cd into some build directory
+$ cmake ../iree -G Ninja \
```

Comment:
```suggestion
cmake -G Ninja -B ../iree-build/ \
```

To build the tensorflow compiler this currently _does_ have to be out of tree. See https://github.com/google/iree/issues/4195.

Up to you how you want to adjust the rest of the doc to match.

Reply:
My command was out-of-tree too! The only difference is what directory we run the cmake command from, but it's out-of-tree in both cases. I switched it to be that way so that we can run all our commands from the build directory, as opposed to having to cd to the build directory below to run the python commands. What do you think?

Reply:
(I have checked these commands locally)

Reply:
Does this block start from the parent directory of the iree git repo? I was reading it as starting within the git repo, which would create `iree/iree-build` and fail. I guess that from the second line highlighted you can see that it had to have been out of tree. Could we maybe add a note that this assumes you're running this from iree's parent dir?

Reply:
oh i see. Yes, It's meant to start from the parent directory of the git repo. ok adding a note.